cmd/manager/main.go

```diff
@@ -36,6 +37,8 @@ var (
 	metricsHost               = "0.0.0.0"
 	metricsPort         int32 = 8383
 	operatorMetricsPort int32 = 8686
+
+	osStat = os.Stat
```

Comment:
This was done so that I can mock it in TestCheckIfRequiredFilesExist in main_test.go. Check [this](TestCheckIfRequiredFilesExist). Let me know if you can suggest any other way to mock os.Stat

Reply:
I would consider a different approach than trying to mock os.Stat. 

This function is dependent on the filesystem. What is your reasoning for a unit test vs an end to end test here?

Reply:
Just talked with @aravindhp and I did not realize that you were directed to use a unit test.

Thinking about this you can alter the checkIfRequiredFilesExist function to make this more unit testable.

Changing the function signature to  `func checkIfRequiredFilesExist(fileList []string) error` allows you to change the files you are looking for dynamically. In your test you can then pass in a some files that are guaranteed to not exist.